Version adkif

adkif opened the pull request:
Cette version marche bien mais j'aurai besoin des données dans les tables paniers et commandes pour faire les tests

alvinBM commented:
> Cette version marche bien mais j'aurai besoin des données dans les tables paniers et commandes pour faire les tests

J'ai créé des données test dans la BD pour vous. 
Great

adkif commented:
Merci

adkif commented:
j'essaie de faire un test mais on me renvoie cette erreur,
la table "paniers" n'existe pas.

`code: 'ER_NO_SUCH_TABLE',
    errno: 1146,
    sqlState: '42S02',
    sqlMessage: "Table 'u591745823_mauto.Paniers' doesn't exist",
    sql: "SELECT `Paniers`.`id`, `Paniers`.`created`, `Paniers`.`modified`, `Paniers`.`deleted`, `Paniers`.`status`, `Paniers`.`client_id`, `Paniers`.`commande_id`, `commande`.`id` AS `commande.id`, `commande`.`created` AS `commande.created`, `commande`.`modified` AS `commande.modified`, `commande`.`deleted` AS `commande.deleted`, `commande`.`status` AS `commande.status`, `commande`.`client_id` AS `commande.client_id`, `commande`.`product_id` AS `commande.product_id`, `commande`.`quantite` AS `commande.quantite` FROM `Paniers` AS `Paniers` INNER JOIN `commandes` AS `commande` ON `Paniers`.`commande_id` = `commande`.`id` WHERE `Paniers`.`id` = '2';"`
